gui/tiletypes.lua

```diff
@@ -7,7 +7,7 @@ local utils = require('utils')
 local guimat = require('gui.materials')
 
 local UI_AREA = {r=4, t=19, w=35, h=35}
```

Comment:
other similar tools use `r=2` and `t=18` -- snug up against the vanilla elements